src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.module.scss

```diff
@@ -50,6 +50,11 @@
       flex-direction: column;
       gap: $spacing-lg;
 
+      // There's a link in the BVD overview content:
+      a {
+        color: $color-blue-50;
+      }
+
```

Comment:
nit: Still need this bit (not sure if the URL removal is permanent or just temporary until we get a newer URL).

Reply:
Permanent, I think: 0e795cc79.